Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -56,6 +56,9 @@ static StyledMarkedText resolveStyleForMarkedText(const MarkedText& markedText,
             style.textStyles.strokeColor = renderStyle->computedStrokeColor();
             style.textStyles.hasExplicitlySetFillColor = renderStyle->hasExplicitlySetColor();
 
+            if (auto* shadowData = renderStyle->textShadow())
+                style.textShadow = ShadowData::clone(shadowData);
```

Comment:
what's the reasoning behind `ShadowData::clone`?

Reply:
I tried numerous ways to copy it over to `style.textShadow` and the only way that worked was `ShadowData::clone`. It's done in other places as well, such as `TextPaintStyle computeTextSelectionPaintStyle` and `StyledMarkedText::computeStyleForUnmarkedMarkedText`.